app/services/data/bank-account-details.js

```diff
@@ -1,10 +1,10 @@
 const config = require('../../../knexfile').extweb
 const knex = require('knex')(config)
 
-module.exports.insert = function (claimId, bankDetailsData) {
+module.exports.insert = function (claimId, bankDetailsDetails) {
```

Comment:
rename var bankAccountDetails
also add check to throw error if called with `(!bankAccountDetails instanceof BankAccountDetails)`